db/schema.rb

```diff
@@ -139,6 +139,7 @@
     t.boolean "active_alert", default: false
     t.text "national_insurance_number"
     t.boolean "trn_auto_verified", default: false
+    t.boolean "admin", default: false
```

Comment:
It was lightly designed at the very start and I implemented it. I don't believe it is linked from anywhere as it's not really fit for purpose. I just left it dormant in case we did want to bring it to life.

Reply:
I mean, do they use a password, do they use a magic link?

Reply:
Oh sorry, they get a 6 digit code via email. Which they have to enter on the next screen. They get a 15 minute window to do this otherwise have to request a new code.